revised/beamswap-proposal.md

```diff
@@ -0,0 +1,330 @@
+
+The rich array of opportunities has attracted a strong, and growing, user base. On average, Beamswap has 3,940 unique monthly users and onboards 1,855 new users per month. To date, 25,977 unique users in total have interacted with the Beamswap dapp in one way or another on the Moonbeam Network.
+
+## ![|602x336](upload://75F6DG9Az6RDbb3OX8cSCJpjsPs.jpeg)
+
+## Beamex Beta
+
+Beamex is the newest perpetual DEX that evolved from Beamswap DeFi Hub and delivers opportunities for trading derivative crypto contracts.
+
+As a decentralized spot and perpetual exchange, Beamex delivers advanced crypto trading to the Moonbeam network, both market order swaps and leverage trading of crypto futures, at low fees and zero price impact.
+
+![|602x339](upload://1FHnZ0saMSo0aa6sui7p9KCd5Br.jpeg)
+
+An in-depth article about Beamex and its core features can be read [here](https://medium.com/beamswap/announcing-beamex-a-decentralized-perpetual-exchange-f8a81b0447cf).
+
+### Beamex Beta first week statistics
+
+![|602x339](upload://6XeJKGFJn4mRwuNFnLmUsrTPR7E.jpeg)
+
+The Beamex Beta is delivered on the BSC Testnet, where users test the exchange features using imaginary tokens. The trade results achieved on Beamex Beta cannot be translated to the production-level Beamex due to factors such as using real money, low risk-taking, and lower TVL. For the Beamswap team, the Beamex Beta represents a test version, and the community should approach it as merely a way to familiarize itself with product features.
+
+## 8 Team Experience
+
+The Beamswap team represents a tight-knit international group of 15+ members, crypto veterans with a strong background and years of experience in Web3 development, DeFi and TradFi, and strategic marketing, working tirelessly to deliver outstanding service, scout new partnership opportunities, and contribute to the evolution of the ecosystem.
+
+Most team members have public names on their social profiles and wish not to be named in the present grant. The Moonbeam Foundation is acquainted with all the key team members and has met several of them at past crypto conferences and events.
+
+The Beamswap team has participated in various roundtable talks, attended many Web3 and crypto events, and held smooth discussions with the Moonbeam ecosystem representatives. Just recently, Beamswap was present at ETHDenver, where Beamex was introduced to the world at the Moonbeam booth.
+
+## 9 Timeline and Milestones for Use of Grant
+
+With upcoming launches of Beamex, the first perpetual exchange on Moonbeam, and Beamswap’s concentrated liquidity v3, both platforms will be in need of boosted liquidity to perform simultaneously and optimally.
+
+The Beamswap Flywheel will leverage all the Beamswap products and consist of the following:
+
+Connected contracts <> Beamswap v2 <> Beamswap v3 <> Beamex
+
+![|602x339](upload://2fsk4Ns2dkTo8tvs0OZnF98lbA7.jpeg)
```

Comment:
Can you also upload the images to the revised/assets/beamswap folder? Otherwise we'll end up with some empty placeholders.